pkgs/development/php-packages/ssh2/default.nix

```diff
@@ -0,0 +1,19 @@
+{ buildPecl, lib, libssh2 }:
+
+buildPecl rec {
+  version = "1.3.1";
+  pname = "ssh2";
+
+  sha256 = "sha256-kJOh+NJNxlg2AnsOI5xQ3o1err+DlrwzMf3TjF1pr9k=";
+
+  buildInputs = [ libssh2 ];
+  configureFlags = [ "--with-ssh2=${libssh2.dev}" ];
+
+  meta = with lib; {
+      changelog = "https://pecl.php.net/package-info.php?package=ssh2&version=${version}";
+      description = "PHP bindings for the libssh2 library";
+      license = licenses.php301;
+      homepage = "https://github.com/php/pecl-networking-ssh2";
+      maintainers = teams.php.members ++ [ maintainers.ostrolucky ];
+    };
```

Comment:
```suggestion
    changelog = "https://pecl.php.net/package-info.php?package=ssh2&version=${version}";
    description = "PHP bindings for the libssh2 library"
    license = licenses.php301;
    homepage = "https://github.com/php/pecl-networking-ssh2";
    maintainers = teams.php.members ++ [ maintainers.ostrolucky ];
  };
```

Reply:
(This is to address the failure of `Checking EditorConfig / tests (pull_request_target) Failing after 2s`)

Reply:
Is there some command I can run locally to fix this automatically?

Reply:
There is no consensus on the codestyle, there are multiple tools:
- [nixfmt](https://github.com/serokell/nixfmt)
- [nixpkgs-fmt](https://github.com/nix-community/nixpkgs-fmt)
- [alejandra](https://github.com/kamadorueda/alejandra)

What I would advise in this case is to get inspiration from what's already existing.